src/java/org/apache/cassandra/repair/autorepair/AutoRepairState.java

```diff
@@ -84,13 +86,22 @@ public abstract class AutoRepairState
     @VisibleForTesting
     protected int skippedTablesCount = 0;
     @VisibleForTesting
+    protected long totalBytesToRepair = 0;
```

Comment:
In future, we can extend by adding expected/actual partitions, token ranges, etc.